src/docs/product/data-management-settings/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,130 @@
+  You can use the `stack_memory` path item to select these regions.
+  Generally they will contain binary data, so be careful with the [data types](../advanced-datascrubbing/#data-types) used to scrub these regions.
+
+  We **do not recommend** scrubbing stack memory since it is required by Sentry to build a readable stack trace for the event.
+  Inadvertently modifying the stack in an undesired way will make this process impossible and the event far less useful.
+  Because of this there is no way to select the stack memory by a more generic selector like a [value type](../advanced-datascrubbing/#value-types), you must select it explicitly e.g. using `stack_memory` or `$minidump.stack_memory` for it to be scrubbed.
+
+  In general, all [data types](../advanced-datascrubbing/#data-types) result in a textual regular expression match on some data.
+  For the stack, this regular expression is matched as a binary UTF-8 regular expression so that any UTF-8 embedded in the binary data will be matched.
+  Additionally the regular expression is applied to any valid UTF-16LE strings found within the binary data.
+
+- The **heap memory**: This is considered to be any memory region which is not used as stack memory.
+  This means these memory regions could include other non-stack regions, such as files that are mapped to the process' memory using  `mmap`.
+  All these memory regions can be selected using the `heap_memory` path item selector.
+  Additionally these regions also have `$binary` as a [value type](../advanced-datascrubbing/#value-types) selector, allowing you to write more generic rules.
+
+  Since Sentry does not require heap memory for extracting stack traces, it is much safer to modify these memory regions.
+
+  Similar to stack memory, all [data types](../advanced-datascrubbing/#data-types) are matched as a binary UTF-8 regular expression as well as in any valid UTF-16LE strings found within the binary memory region.
+
+- A minidump also contains a list of **code modules**,  which are loaded at the time the minidump is created.
+  This module list contains information about each module, including two fields that may contain file paths: the path of the code file and the path of the associated debug file.
+
+  Since these fields are pathnames, they have the potential to contain a user's home directory.
+  As a result, it makes sense to apply the *Usernames in filepaths* [data type](../advanced-datascrubbing/#data-types) to these fields.
+  The fields can be selected individually by their respective field names:
+
+  * `code_file`
+  * `debug_file`
+
+  Both fields have a [value type](../advanced-datascrubbing/#value-types) of `$string`. Any modification will exclude the basename of the pathnames to ensure the event processing pipeline can still operate correctly on the minidump.
+
+  This example is how the UI displays a rule to scrub these fields:
```

Comment:
suggest:
This example illustrates how a rule to scrub these fields displays in sentry.io: